### ISSUE TITLE
Return values of abjad.persist(client).as_png()

### ISSUE THREAD

**Issue by nivlekp** [Abjad/abjad#1173]
Hi all,

__TL;DR__: `abjad.persist(client).as_png()` indicates unsuccessful even though output files are generated successfully.

I am not sure if this is an issue, but I was trying to run `abjad.persist(client).as_png()` as illustrated in https://abjad.github.io/api/abjad/system/PersistenceManager.html?highlight=persistencemanager#abjad.system.PersistenceManager.PersistenceManager.as_png

After running
```
>>> staff = abjad.Staff()
>>> container = abjad.Container("c'4 d'4 e'4 f'4")
>>> command = abjad.LilyPondLiteral(r"reak", "after")
>>> abjad.attach(command, container[-1])
>>> staff.extend(200 * container)
```
then
```
>>> result = abjad.persist(staff).as_png()  
>>> for x in result[0]:  
...     x
```
gave me
```
'/Users/tszkiupang/.abjad/output/0009-page1.png'
'/Users/tszkiupang/.abjad/output/0009-page2.png'
'/Users/tszkiupang/.abjad/output/0009-page3.png'
```
and I had no problem opening the output files.

However if I run
```
>>> result, _, _, success = abjad.persist(staff).as_png()
>>> for x in result:
...    x
```
that gave me
```
'/Users/tszkiupang/.abjad/output/0010-page1.png'
'/Users/tszkiupang/.abjad/output/0010-page2.png'
'/Users/tszkiupang/.abjad/output/0010-page3.png'
```
Again I had no trouble at all opening the files, but `success` is now `False`, which seems weird to me since the output files were generated successfully.

### Some background context
I was trying to run `abjad.show(staff)` in Jupyter Notebook and this seems to be the problem...

**Comment by trevorbaca** [comment]
The updated 3.2 version looks like this:
```
staff = abjad.Staff()
container = abjad.Container("c'4 d'4 e'4 f'4")
command = abjad.LilyPondLiteral(r"reak", "after")
abjad.attach(command, container[-1])
containers_ = []
for i in range(200):
    container_ = abjad.mutate.copy(container)
    containers_.append(container_)
staff.extend(containers_)
result = abjad.persist.as_png(staff, "/Users/trevorbaca/Desktop/test.png")
for x in result:
    print(x)
('/Users/trevorbaca/Desktop/test-page1.png', '/Users/trevorbaca/Desktop/test-page2.png', '/Users/trevorbaca/Desktop/test-page3.png')
0.09853315353393555
2.011029005050659
False
```

Still produces all three pngs correctly, still returns `success` as false.

**Comment by nivlekp** [comment]
Coming back to this after 9 months, now I realized this is just a typo on my part, sorry.
The problematic line is `command = abjad.LilyPondLiteral(r"reak", "after")
`success` should be `False` because lilypond actually failed but somehow still render the image anyway (not correctly though).
The following code should have no problem at all:
```python
>>> staff = abjad.Staff()
>>> container = abjad.Container("c'4 d'4 e'4 f'4")
>>> command = abjad.LilyPondLiteral(r"\break", "after")
>>> abjad.attach(command, container[-1])
>>> containers_ = []
>>> for i in range(200):
...     container_ = abjad.mutate.copy(container)
...     containers_.append(container_)
>>> staff.extend(containers_)
>>> result = abjad.persist.as_png(staff, "~/.abjad/test.png")
>>> for x in result:
...     print(x)
('/Users/tszkiupang/.abjad/test-page1.png', '/Users/tszkiupang/.abjad/test-page2.png', '/Users/tszkiupang/.abjad/test-page3.png', '/Users/tszkiupang/.abjad/test-page4.png', '/Users/tszkiupang/.abjad/test-page5.png', '/Users/tszkiupang/.abjad/test-page6.png', '/Users/tszkiupang/.abjad/test-page7.png', '/Users/tszkiupang/.abjad/test-page8.png', '/Users/tszkiupang/.abjad/test-page9.png', '/Users/tszkiupang/.abjad/test-page10.png', '/Users/tszkiupang/.abjad/test-page11.png', '/Users/tszkiupang/.abjad/test-page12.png', '/Users/tszkiupang/.abjad/test-page13.png', '/Users/tszkiupang/.abjad/test-page14.png', '/Users/tszkiupang/.abjad/test-page15.png')
0.09218907356262207
2.523041009902954
True
```

**Comment by trevorbaca** [comment]
Ah ok! So nothing to fix for 3.3? @nivlekp should we close the issue?

**Comment by nivlekp** [comment]
Yes, there's nothing to fix. I'm closing it now.
